nipoppy/data/hpc_templates/slurm/slurm.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+#SBATCH --output=time_%A_%a.out            # Output filename pattern with job and array task ID
+#SBATCH --job-name={{ job_name }}          # Job name
+#SBATCH --account={{ account_name }}       # Account name
+#SBATCH --chdir={{ working_directory }}    # Change to the specified directory
```

Comment:
It can be removed.
I was using it as a flexible way of specifying where the log files are dumped. I've now specified the log dump path using --output and --error like this:
`#SBATCH --output={{ dataset_root }}/logs/hpc/%x_%A_%a.out`
I'm now specifying this dataset_root as self.dpath_root in the pysqa call in workflows/pipeline.py